src/components/Dashboard /PartnerData/index.js

```diff
@@ -59,6 +59,7 @@ function PartnerDashboard() {
           position: toast.POSITION.BOTTOM_RIGHT,
           autoClose: 2500,
         });
+        setFlag(!flag);
```

Comment:
@Poonam-Singh-Bagh Instead of setting a flag here, can we not set the state of the partner's meraki/app link (the one which just got created) and re-render the component based on this setState?